app/View/Label.php

```diff
@@ -49,6 +49,10 @@ public function render(callable $callback = null)
                 ->with('count', $this->data->get('count'));
         }
 
+        if ($template === null) {
+            throw new \UnexpectedValueException('Template is null.');
```

Comment:
It seems like this would result in a 500 since we're not catching the exception higher up (right?)

Reply:
this is prior to doing anything with the template other than doing an eloquent find()

 ```
$template = LabelModel::find($settings->label2_template);

        // If disabled, pass to legacy view
        if ((!$settings->label2_enable)) {
            return view('hardware/labels')
                ->with('assets', $assets)
                ->with('settings', $settings)
                ->with('bulkedit', $this->data->get('bulkedit'))
                ->with('count', $this->data->get('count'));
        }

        if ($template === null) {
            throw new \UnexpectedValueException('Template is null.');
        }

        $template->validate();
```

Reply:
But if `$template` is null then the user will end up seeing a hard-500 because we're not catching that exception. Maybe we shouldn't throw the exception but send them somewhere else with an error?

Reply:
Ok, i changed it to a try catch 👍